osu.Game/Screens/Edit/EditorBeatmap.cs

```diff
@@ -89,6 +89,8 @@ public partial class EditorBeatmap : TransactionalCommitComponent, IBeatmap, IBe
 
         public BindableInt PreviewTime { get; }
 
+        public Bindable<bool> AdjustNotesOnOffsetBPMChange { get; } = new Bindable<bool>(false);
```

Comment:
I'm not sure `EditorBeatmap` is the correct home for this setting. Once we start to persist it it should probably be removed.

Reply:
I did wince at that slightly but it did make *some* sense to me as a place to put "per-edit-session" settings. Do you think it just should be user global instead?